drivers/axi_core/axi_dmac/axi_dmac.c

```diff
@@ -303,10 +303,6 @@ int32_t axi_dmac_init(struct axi_dmac **dmac_core,
 	dmac->direction = init->direction;
 	dmac->flags = init->flags;
 	dmac->transfer_max_size = -1;
-	dmac->big_transfer.address = 0;
```

Comment:
the lines were deleted by mistake, i guess? You are adding `dmac->big_transfer.transfer_done = false;` in previous commit

Reply:
The deletion is not a mistake. The change could have been in one commit, but I thought it would have been harder to explain what I tried to do so I separated it. These lines are not needed anymore because `calloc` already initializes them to 0. Now I realize it might be easier to just drop the first commit and just solve the bug as a byproduct of the second. What do you think? Should we even mention the bug anymore since it can be solved sneakily?

Reply:
Got it! Adding and then removing the same line got me confused. Well, I would just do the `calloc` and remove the explicit initializations with 0 in a single commit. And mention also the transfer issue that was fixed in the commit description.